career/interviewing/technical-interviewing.md

```diff
@@ -57,9 +57,9 @@ Walk through several example questions and demonstrate successful and unsuccessf
 Look through the materials sections of this guide and try a variety of different challenges. Do several questions for each website and then move on to the next one. Get comfortable with questions that feel and look unfamiliar.
 
 ### Challenge
-- Build a small React app that fetches some data from an API. Try finding something at [Any API](https://any-api.com/?tag=open%20data) for example to provide an API service to use.
-- When the participant is able to fetch data and render it in their own React app have them manipulate the data in some way. For example, make every third word of a paragraph all caps.
-
+- Write a Binary Search program using any programming language.
```

Comment:
I suggest to either replace this question or modify it a little bit because binary search is not asked so much in the interviews.

Reply:
How about this one?
```suggestion
- Given an unsorted array of size N of positive integers. One number 'A' from set {1, 2, …N} is missing and one number 'B' occurs twice in the array. Find these two numbers.
```

Reply:
Yeah, this one is better.